packages/cadl-ranch-specs/http/server/versions/not-versioned/main.tsp

```diff
@@ -0,0 +1,38 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+
+using TypeSpec.Http;
+
+/**
+ * Illustrates not-versioned server.
+ */
+@scenarioService("/server/versions/notVersioned")
```

Comment:
`scenarioService` would sets a 1.0.0 to service version
https://github.com/Azure/cadl-ranch/blob/main/packages/cadl-ranch-expect/src/decorators.ts#L228-L231

I am not sure whether emitter would/should take this as api-version.

We currently seems having no way in typespec to tell us whether this is Azure or non-Azure.

Reply:
This link you provided is setting service version, not related to versioned. So we are still not versioned.

> I am not sure whether emitter would/should take this as api-version.

Definitely it should not, and that is exactly what this test is testing, i.e., anything in `@service` should not be taken as version.

Reply:
I do not object to it. But please do have other languages give their opinion about whether they are on same page on this `@server(version=..).

I agree that non-Azure should definitely not take this. I am not exactly sure whether Azure should take this or not (but inclined to say not).

Reply:
I talked with @timotheeguerin long time ago about this. Everything in `@server(version=..)` is just metadata and that's it. All the versions should go in `@versioned`. Now @m-nash agrees with it too. Further, it applies to both azure and non-azure.

> Michael's position is that we should do at little as possible on this if (azure) {} else {}

A flag doesn't mean we will have such code. In .net we will have default non-azure behavior, and if the flag set we just override the default code to generate azure code (if the flag says azure). But how other languages do this I've no ideas.